nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -547,8 +548,8 @@ define([
                 const appSpecInfo = appSpecs[inputs[fileType].appId].full_info;
                 const appInfo = {
                     app_id: appSpecInfo.id,
-                    tag: 'release',
-                    version: appSpecInfo.ver,
+                    tag: appSpecInfo.tag || 'release',
+                    version: appSpecInfo.git_commit_hash,
```

Comment:
include the app tag and use the git commit hash instead of the text version string to enable the running of apps in dev / beta state.